### PR TITLE
Add Governing Board member bios

### PR DESCRIPTION
**Pull request by wipfli**
This draft of the Governing Board member bios is taken from the self-nominations at https://github.com/maplibre/maplibre/discussions/78. Please adjust your text according to your needs.

@birkskyum @HarelM @lseelenbinder @klokan @nyurik

### CODE REVIEW

File: content/about.html [hunk 1]
```diff
@@ -22,12 +22,276 @@ <h2>About</h2>
...
+  <p>
+    I generally enjoy participating in open source projects from the development
+    side of things.
+  </p>
+  <p>
+    I'm currently employed at Dell as a group leader and previously done roles
+    as a team leader and architect in the Semi-conductor industry.
+  </p>
+  <p>Feel free to check out my linked-in profile here:</p>
+  <p>https://www.linkedin.com/in/harel-mazor-1298b139/</p>
```
Comment (HarelM) [inline content/about.html:107]: `<a href=...`

Comment (HarelM) [reply content/about.html:107]: Also for the other links

File: content/about.html [hunk 2]
```diff
@@ -22,12 +22,276 @@ <h2>About</h2>
...
+  <img src="https://avatars.githubusercontent.com/u/3269297?v=4" width="200" />
+
+  <p>Hi everyone!</p>
+  <p>
+    My name is Harel Mazor and I would like to nominate myself for the Governing
+    Board.
+  </p>
+  <p>
+    I'm currently maintaining maplibre-gl-js (You might have seen some of my
+    annoying comments if you have submitted a PR there :-)).
```
Comment (HarelM) [inline content/about.html:82]: replace with :-) with 😊

**Comment by HarelM** [comment]
I've updated my. Thanks!

**Comment by maxammann** [comment]
I think we should convert this to markdown. I'll probably create a PR to convert everything to markdown after this PR.

**Comment by wipfli** [comment]
This is ready to be published from my point of view...

**Review by HarelM — APPROVED** [review]
Push it, we can always change it more in the future.

**Pull request merged**